websites/C/Cuevana 3/presence.ts

```diff
@@ -126,7 +125,10 @@ presence.on("UpdateData", async () => {
 				presenceData.smallImageText = videoPaused
 					? strings.pause
 					: strings.play;
-				if (!videoPaused) presenceData.endTimestamp = endTimestamp;
+				if (!videoPaused) {
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestampsfromMedia(video);
```

Comment:
`video` doesn't exist. Some data is retrieved from an iframe. Consider adding a `startTimestamp` global variable to the original code instead.